### ISSUE TITLE
Select projects and collect notes for initial swagmap projects

### ISSUE THREAD

**Issue by usernamenumber** [tunapanda/swag-old#12]
Someone at the institute (@jaylarson85?) should select the project(s) we most want to be able to use for a demo of Swag. For each project, we'll need all the details that should go into a swagmap for it, including:
1. Descriptions of each point that should go on each map 
   - I'm unclear from our discussion whether the idea is for the points to be objectives within a project or projects within a curriculum, or something in between
2. The student notes for each point 
   - In other words, the information students should see when they click that point on the map, or a link to that information

**Comment by usernamenumber** [comment]
@jaylarson85 proposes the following for initial swagmaps:
1. Intro to Scratch (good to show coding work)
2. Business basics (quite note-heavy)
3. ADDIE (model for creating content - will ensure we create these swagmaps in an intelligent manner)

**Comment by usernamenumber** [comment]
@jaylarson85, do we have detailed lesson plans / exercises for these, from which to start generating nodes/swagifacts for the map? If so, can you provide links?

**Comment by jaylarson85** [comment]
Working on getting those together.
